src/App.js

```diff
@@ -1,13 +1,85 @@
-import React from 'react';
+import React from "react";
+import TodoList from "./components/TodoComponents/TodoList";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import Search from "./components/TodoComponents/Search";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      todoData: [],
+      todoDisplayData: [],
```

Comment:
It might be easier to use a display boolean on the main todo array rather than duplicating, but up to you !